sfcasts/crud-controller.md

```diff
@@ -1,171 +1,90 @@
+
+We'll go to `DashboardController` and it doesn't matter where but I'll go down to the bottom. I'm going to go to "Code Generate" or "cmd + N" on a Mac, and go to "Override Methods..."" and override `configureCrud`. The first thing we want to do is call `paren::configureCrud`. So it can instantiate that object.
+
+Oh! but before we do, we actually want to go to "Override Methods..." and select `configureActions`. Now we want to call the parent method that can create the actions object and set up all of those default actions for us. To do that, add an action to our index page. We can use that CRUD class call on page index.
+
+```terminal
+->add(Crud::PAGE_INDEX)
+```
+
+ We're in, and we can actually use a constant for this as well.
+ 
+ ```terminal
+Action::DETAIL
+ ```
+
+So on the index page, I want to add the action and then the `DETAIL` action. The result of this, when we refresh the `QuestionCreditController`, is that we have this "Show" link that goes to the `DETAIL` action, and we can go to any section. 
+
+And just like that, we've modified every single credit controller in the system! Easy peasy! *But*, since this entity is so simple, let's disable the `DETAIL` action for just this topic section. To do that, open up `TopicCrudController.php`, and just like before, we'll go to "Code Generate" or "cmd + N" on a Mac and go to "Override Methods..." and override that same `configureActions`.
+
+By the time this method is called, it will pass us. The actions of that was already set up by our dashboard, so it will already have the `DETAIL` action enabled for the index page. But *now*, we can control that by saying:
+
+```terminal
+->disable(Action::DETAIL)
+```
+
+We're gonna talk more about the actions configuration, but these are the main things that you can do inside of them: You can add a new action to a page, *or* you can just completely disable an action globally for all of your pages. As soon as we refresh here, our `DETAIL` action is gone, but if we go to our other sections, it's *still* there.
+
+The big takeaway here is that everything is going through our `DashboardController`, which means that we can configure things on a dashboard level, which will apply to all of our CRUD, *or* we can configure things for one specific CRUD. The fact that all of the CRUD controllers go through this `/avenue`, where all has one other effect related to security, means that all of our controllers are already secure. That's thanks to our `access_control`.
```

Comment:
I was unsure about `/avenue` because the "/" was already there. Is that actually referring to anything specific or was that slash unintended?

Reply:
The part of that sentence "where all has one other effect" kind of threw me off. Would "where each" instead of "where all" sound better?

Reply:
> I was unsure about /avenue because the "/"

I said this FAST. But it should be "/admin URL" - I'm quickly showing the address bar, which starts with `/admin`.

> The part of that sentence "where all has one other effect" kind of threw me off. Would "where each" instead of "where all" sound better?

The original audio is:

> The fact that all of the CRUD controllers go through this `/admin` URL has one other effect related to security.

So I think this is just hard-to-understand audio / bad-automated transcription causing the issue.